### PR TITLE
[CUBRIDQA-33] add space for license stmt and unify the names of category

### PR DESCRIPTION
**Pull request by supler**
@fanzaiqiang 

add space for license stmt and unify the names of category

### CODE REVIEW

File: CTP/README.md [hunk 1]
```diff
@@ -152,13 +152,13 @@ The section guides users to quickly start sql test with CTP, for the more catego
...
 			```
 			JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
 			```
 			* Check out CTP and configure CTP/conf/shell.conf for testing
 			```
-			# Test instance information. For ha shell, relatedhosts must be configured for slave
+			# Test instance information. For HA SHELL, relatedhosts must be configured for slave
 			# The property values configured will be updated into CUBRID conf for testing
 			env.instance1.ssh.host=192.168.1.10
 			#env.instance1.ssh.relatedhosts=192.168.1.11
```
Comment (jinmeihua) [inline CTP/README.md]: #env.instance1.ssh.relatedhosts=192.168.1.11
Please add a space after "#".

**Comment by jinmeihua** [comment]
@supler 

According to Mr. Kim's suggestions, 
I also reviewed and commented on https://github.com/CUBRID/cubrid-testtools/pull/283. 
Please also handle them. 
Thank you.

File: CTP/README.md (hunk 1 above)
Comment (xkyu) [inline CTP/README.md]: Please remove verbose example like below paragraphs too:
```
 			JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
```

**Comment by jinmeihua** [comment]
@supler 

OK, please go ahead.
Thank you. :+1:

**Pull request merged**